utils/http_utils.go

```
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+var (
+	Client HTTPClient = &http.Client{}
+)
+
+type HttpResponse struct {
+	Body []byte
+	Headers http.Header
+	StatusCode int
+}
+
+// It send an http post to the endpoint signing the body with the worker's private key
+func Post(workerId string, body interface{}, endpoint string) *HttpResponse{
+	requestBody, err := json.Marshal(body)
```

any reason to ignore the error?